docs/guides/operator-private-gcp-confidential-space.md

```diff
@@ -500,7 +501,8 @@ import AttestFailure from '/docs/snippets/_private-operator-attest-failure.mdx';
 
 <AttestFailure />
 
-## Upgrading
+
+### Upgrading
```

Comment:
Change the title level because it should be wrapped under Task:
<img width="1004" alt="image" src="https://github.com/user-attachments/assets/9e7cf0f4-f46e-4abf-a912-c0e47080494a">

Reply:
Good point, I think `Upgrading` should be at the same title level as `Tasks`. @thomasm-ttd Can you confirm this?

Reply:
I think we should drop Tasks and the paragraph after that. 
So looks like this:
## Running the Health Check
### Private Operator Attestation Failure
## Upgrading
## Scraping Metrics